dvc/repo/plot.py

```diff
@@ -0,0 +1,66 @@
+import json
+import logging
+
+from dvc.plot import DefaultTemplate
+
+logger = logging.getLogger(__name__)
+
+
+class PageTemplate:
```

Comment:
Seems like class is not really needed here. You are using it as a function anyways. Or am I missing something?